froi/core/dataobject.py

```diff
@@ -786,14 +786,40 @@ def __init__(self, surf_path, offset=None):
             applied. If != 0.0, an additional offset will be used.
 
         """
-        self.surf = SurfaceDataset(surf_path, offset)
-        self.name = self.surf.name
+        # self.surf = SurfaceDataset(surf_path, offset)
+        type = 'white'
```

Comment:
I would fix this by using 'surf_type' to replace 'type', and I'm sorry for this fault...